render/extensions/extensions.go

```diff
@@ -158,6 +172,12 @@ func StubTemplateFunc(...string) (string, error) {
 	return "", nil
 }
 
+func newStub(returnVal interface{}) func(...string) (interface{}, error) {
```

Comment:
so this is basically just an `always`, right? Just trying to make sure I understand the semantics here.